### PR TITLE
DCOS-12335: Fix pod scaling

### PR DESCRIPTION
**Pull request by wavesoft**
This PR adds the missing scaling reducers for the multi-container form.

### CODE REVIEW

File: plugins/services/src/js/reducers/serviceForm/MultiContainerScaling.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+  JSONParser(state) {
+    if (state.scaling == null) {
+      return [];
+    }
+
+    let transactions = [];
+    if (state.scaling.instances) {
+      transactions.push(
+        new Transaction(['instances'], state.scaling.instances));
```
Comment (nLight) [inline plugins/services/src/js/reducers/serviceForm/MultiContainerScaling.js]: would you add a new line before `);` too?

File: plugins/services/src/js/reducers/serviceForm/MultiContainerScaling.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    }
+    if (state.scaling.kind) {
+      transactions.push(
+        new Transaction(['scaling', 'kind'], state.scaling.kind));
```
Comment (nLight) [inline plugins/services/src/js/reducers/serviceForm/MultiContainerScaling.js]: and here

**Comment by wavesoft** [comment]
The error message has to go to the `raml-validator` package. I agree that we need a better error there.

**Comment by wavesoft** [comment]
retest this please

**Pull request merged**